lib/src/components/locale/shadcn_localizations.dart

```diff
@@ -736,3 +711,879 @@ class DefaultShadcnLocalizations extends ShadcnLocalizations {
   @override
   String get colorLightness => 'Lum';
 }
+
```

Comment:
I just realized that this file will have thousands of lines when we add a new locale, can we separate this into `korean_shadcn_localizations.dart` and `japanese_shadcn_localizations.dart`?

Reply:
I push the file separately